src/app/( Test Pages )/login/action.ts

```
@@ -0,0 +1,56 @@
+  const clinicId = process.env.NEXT_PUBLIC_CLINIC_ID || "";
+  try {
+    const profiles = await api.auth.getClinicProfiles({ clinicId });
+    return profiles;
+  } catch (error) {
+    console.error("Failed to fetch profiles:", error);
+    return [];
+  }
+}
+
+// Login action function
+export async function loginAction(state: any, formData: FormData) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)